### PR TITLE
Document SafeEvpPKeyHandle

### PR DESCRIPTION
**Pull request by bartonjs**

### CODE REVIEW

**Comment by bartonjs** [comment]
cc: @carlossanlop

File: xml/System.Security.Cryptography/SafeEvpPKeyHandle.xml [hunk 1]
```diff
@@ -67,9 +75,11 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Creates another instance of this type which has an independent lifetime, but tracks the same resource.</summary>
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography/SafeEvpPKeyHandle.xml]:
```suggestion
        <summary>Creates another instance of this type which has an independent lifetime but tracks the same resource.</summary>
```

File: xml/System.Security.Cryptography/SafeEvpPKeyHandle.xml [hunk 2]
```diff
@@ -67,9 +75,11 @@
...
+        <returns>Another instance of this type which has an independent lifetime, but tracks the same resource.</returns>
```
Comment (rpetrusha) [inline xml/System.Security.Cryptography/SafeEvpPKeyHandle.xml]:
```suggestion
        <returns>Another instance of this type which has an independent lifetime but tracks the same resource.</returns>
```

**Comment by carlossanlop** [comment]
@rpetrusha @mairaw The suggestions were applied and the build passed. Is this good to merge?

**Comment by rpetrusha** [comment]
Merging now, @bartonjs and @carlossanlop.

**Pull request merged**